server.js

```diff
@@ -0,0 +1,77 @@
+
+let bodyParser = require('body-parser');
+let express = require('express');
+
+let app = express();
+let port = 8080;
+let buzzWords = [];
+let justBuzzWords = [];
+let score = 0;
+
+app.use(express.static('public'));
+app.use(bodyParser.urlencoded({ extended: true }));
+
+
+app.get('/buzzwords', function (req, res) {
+  res.json({buzzWords: buzzWords});
+});
+
+
+app.post('/buzzword', function (req, res) {
+  if(typeof req.body.buzzWord==='string' && typeof parseInt(req.body.points)==='number' && buzzWords.length < 5 && justBuzzWords.indexOf(req.body.buzzWord) === -1){
```

Comment:
I'm not sure why you have buzzWords.length < 5. I don't think the instructions had a limit on the number of buzzWords a user could have?